common/src/paused_program_storage.rs

```diff
@@ -40,7 +35,8 @@ impl From<(BTreeSet<WasmPage>, H256, MemoryMap)> for Item {
     fn from(
         (allocations, code_hash, mut memory_pages): (BTreeSet<WasmPage>, H256, MemoryMap),
     ) -> Self {
-        let remaining_pages = memory_pages.split_off(&GearPage::from(SPLIT_COUNT));
+        let split_page: GearPage = WasmPage::from(MAX_WASM_PAGE_COUNT / 2).to_page();
```

Comment:
@gshep please see this place. I cannot understand why did we need MAX_WASM_PAGE_COUNT here. May be we can just use some in-depended constant WasmPage(256) ?

Reply:
That is actually why we cannot increase memory limit. Will discuss it